glide.yaml

```diff
@@ -23,8 +23,9 @@ import:
 - package: k8s.io/client-go
   version: v2.0.0-alpha.1
 - package: github.com/giantswarm/awstpr
-  repo: git@github.com:giantswarm/awstpr.git
-  version: 3d98b40a2bc399a6a20f3b19c634d56d99a69e81
+  version: 99da9c8f8583749d0fef89d14b3cfa420e13ebc1
+- package: github.com/giantswarm/clustertpr
+  version: 16f7b5230a6850c1b7b62bad4f6002607f8ad3c2
```

Comment:
When `awstpr` is pinned there should be no reason to pin the cluster TPR itself. This repo should be pinned inside the `awstpr` repo.